Winget-AutoUpdate-Install.ps1

```diff
@@ -113,7 +117,7 @@ param(
 
 <# APP INFO #>
 
-$WAUVersion = "1.16.4"
+$WAUVersion = "1.16.5"

```

Comment:
As a new feature, we could even go for a 1.17.0 as pre-release ?

Reply:
It's not my project anymore 😅 so anyone can do what he thinks good for all of us 😊

Reply:
@KnifMelti Do you have an opinion on this? I think 1.17.0 as a pre-release makes more sense as it is a new feature.

Reply:
I beg to differ @Romanitho **Owner**!
It's very much **your** project and we're all very grateful that you took the time to initiate and code the very foundation and higher (in my view) functions/layouts we all (I specifically) depend upon ❤️
This came in a time when it was much needed and is now a part of many users **install-and-forget** applications that just works, both in corporate environments and in private.
So, don't you now go quitting on us!

But, compared to your new Daddy life it's a minor/smaller thing 😉